examples/basics/visuals/mesh.py

```diff
@@ -83,7 +83,8 @@ def on_draw(self, ev):
         gloo.set_viewport(0, 0, *self.physical_size)
         gloo.clear(color='black', depth=True)
         for mesh in self.meshes:
-            mesh.draw(mesh.tr_sys)
+            mesh._prepare_transforms(mesh)
```

Comment:
don't call `_prepare_transforms`; try copying the `on_resize`method from other examples.